src/data_0_10.rs

```diff
@@ -138,22 +150,30 @@ impl TfhersVersion for V0_9 {
         let dir = Self::data_dir().join(HL_MODULE_NAME);
         create_dir_all(&dir).unwrap();
 
-        let config = tfhe_0_9::ConfigBuilder::with_custom_parameters(HL_CLIENTKEY_TEST.parameters)
+        let config = tfhe_0_10::ConfigBuilder::with_custom_parameters(HL_CLIENTKEY_TEST.parameters)
             .enable_compression(VALID_TEST_PARAMS_TUNIFORM_COMPRESSION.into())
             .build();
-        let hl_client_key = ClientKey::generate(config);
+        let (hl_client_key, hl_server_key) = tfhe_0_10::generate_keys(config);
         let compressed_server_key = CompressedServerKey::new(&hl_client_key);
 
         store_versioned_test!(&hl_client_key, &dir, &HL_CLIENTKEY_TEST.test_filename);
+        store_versioned_test!(&hl_server_key, &dir, &HL_SERVERKEY_TEST.test_filename);
```

Comment:
I think you store the same test twice, you should not enable compression for the first server key

Reply:
if I was storing it twice I would get an error for the filname though as I added a panic for it ?

what do you mean by storing it twice here ? Same key with same config ?

Reply:
ah yeah indeed, the server_key has compression enabled for both, will update